src/app/components/Tab/index.tsx

```diff
@@ -1,41 +1,33 @@
 import React from 'react';
-import styled, { css } from 'styled-components/macro';
-import { Text } from '@blueprintjs/core';
+import classNames from 'classnames';
 
-interface Props {
-  text: string;
+type TabProps = {
+  text: React.ReactNode;
   active: boolean;
+  disabled?: boolean;
   onClick: () => void;
-}
+  className?: string;
+};
 
-export function Tab(props: Props) {
-  return (
-    <StyledTab active={props.active} onClick={() => props.onClick()}>
-      <Text ellipsize>{props.text}</Text>
-    </StyledTab>
-  );
-}
-
-interface StyledProps {
-  active: boolean;
-}
-const StyledTab = styled.button.attrs(_ => ({
-  type: 'button',
-  className: 'btn hover:tw-text-gray-9',
-}))`
-  color: var(--sov-white);
-  padding: 5px 10px;
-  background: transparent;
-  font-size: 1.125rem;
-  font-weight: 100;
-  font-family: Montserrat;
-  text-transform: none;
-  ${(props: StyledProps) =>
-    props.active &&
-    css`
-      font-weight: 400;
-      &:hover {
-        color: var(--white);
-      }
-    `}
-`;
+export const Tab: React.FC<TabProps> = ({
+  text,
+  active,
+  onClick,
+  disabled,
+  className,
+}) => (
+  <span
```

Comment:
Can you please make this into a button, so it is focus-able and shows up on screen readers.